### ISSUE TITLE
"window is not defined" in SSR/Sapper when using NavbarToggler/Collapse

### ISSUE THREAD

**Issue by RKasinCDRK** [bestguy/sveltestrap#67]
I've read through a few issues already and there were multiple mentions of importing the source directly for components when using Sapper and that fixing people's issues when combining SSR and NavbarToggler/Collapse. Unfortunately, that did not solve it for me.

It did allow me to render the Navbar, however, when adding NavbarToggler/Collapse to the picture, my app seems to need the window context. Could it be that this is on purpose? Could it be servers-side rendered or is this an unfortunate bug in the framework?

My code is the simple template from the sveltestrap demo for Navigation.
```
<script>
  import Collapse from "sveltestrap/src/Collapse.svelte";
  import Navbar from "sveltestrap/src/Navbar.svelte";
  import NavbarToggler from "sveltestrap/src/NavbarToggler.svelte";
  import NavbarBrand from "sveltestrap/src/NavbarBrand.svelte";
  import Nav from "sveltestrap/src/Nav.svelte";
  import NavItem from "sveltestrap/src/NavItem.svelte";
  import NavLink from "sveltestrap/src/NavLink.svelte";

  let isOpen = false;

  function handleUpdate(event) {
    isOpen = event.detail.isOpen;
  }
</script>

<Navbar color="light" light expand="md">
  <NavbarBrand href="/">sveltestrap</NavbarBrand>
  <NavbarToggler on:click={() => (isOpen = !isOpen)} />
  <Collapse {isOpen} navbar expand="md" on:update={handleUpdate}>
  <Nav class="ml-auto" navbar pills>
    <NavItem>
      <NavLink href="." active>Components</NavLink>
    </NavItem>
    <NavItem>
      <NavLink href="/">GitHub</NavLink>
    </NavItem>
  </Nav>
  </Collapse>
</Navbar>
```

**Comment by bestguy** [comment]
Hi @RKasinCDRK , thanks for the heads up, I'll take a look. The Svelte 3 SSR situation is very confusing to me as I see different configurations for this, and not totally clear what the official way to support this is.  Sapper seems to still be developing too..

**Comment by bestguy** [comment]
Hi again @RKasinCDRK - can you please try sveltestrap@3.2.6-0 and let me know if this corrects your issue ?  This is a prerelease that defensively checks for `window`.

You'll need to install via:
`npm install sveltestrap@3.2.6-0`

**Comment by RKasinCDRK** [comment]
Hi @bestguy, thank you for your fast answer!

It did indeed solve the problem entirely for me.

Thank you very much for the help, I'll close the isse.

**Comment by bestguy** [comment]
Great thanks, I'll publish 3.2.6 today, which you can use instead of prerelease.

**Comment by bestguy** [comment]
Published in [v3.2.6](https://github.com/bestguy/sveltestrap/releases/tag/v3.2.6)
